test/Microsoft.AspNetCore.Mvc.IntegrationTests/ModelBindingTestHelper.cs

```diff
@@ -77,7 +89,7 @@ public static class ModelBindingTestHelper
 
             return new ParameterBinder(
                 metadataProvider,
-                GetModelBinderFactory(metadataProvider, options),
+                new ModelBinderFactory(metadataProvider, options, services),
```

Comment:
Check if `GetModelBinderFactory(...)` is still used. Remove it if not.

Reply:
Yeah, its used in `ParameterBinderExtensions.cs` file